pkg/libvmi/storage.go

```diff
@@ -62,7 +62,15 @@ func WithContainerSATADisk(diskName, imageName string, diskOpts ...DiskOption) O
 func WithPersistentVolumeClaim(diskName, pvcName string, diskOpts ...DiskOption) Option {
 	return func(vmi *v1.VirtualMachineInstance) {
 		addDisk(vmi, newDisk(diskName, v1.DiskBusVirtio, diskOpts...))
-		addVolume(vmi, newPersistentVolumeClaimVolume(diskName, pvcName))
+		addVolume(vmi, newPersistentVolumeClaimVolume(diskName, pvcName, false))
+	}
+}
+
+// WithHotplugPersistentVolumeClaim specifies the name of the hotpluggable PersistentVolumeClaim to be used.
+func WithHotplugPersistentVolumeClaim(diskName, pvcName string, diskOpts ...DiskOption) Option {
```

Comment:
Not a big deal since the boolean is only internal but we could craft some magic to make
`WithPersistentVolumeClaim(diskname, pvcName, WithHotplug())` a thing

Reply:
Good idea but `DiskOption` takes a disk not a volume

Reply:
Yeah I think you'd have to do something ugly with `args ...interface{}` but definitely not for this PR